bin/cli.js

```diff
@@ -148,7 +148,7 @@ program.command('create')
         }
       } catch (err) {
         log.trace(err)
-        log.error('An error occurred while processing user-provided CDXJ indices.')
+        log.warning('An error occurred while processing user-provided CDXJ indices.')
```

Comment:
```suggestion
        log.warn('An error occurred while processing user-provided CDXJ indices.')
```

For console API compatibility